core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -269,6 +269,20 @@ private[spark] object Utils extends Logging {
     file.setExecutable(true, true)
   }
 
+  def moveToTrashIfEnabled(
```

Comment:
Please add function description like the other function in this Utils class.

Reply:
done